cli/commands/backend/migrate/migrate.go

```diff
@@ -0,0 +1,37 @@
+// Package migrate provides the ability to bootstrap remote state backend.
+package migrate
+
+import (
+	"context"
+
+	"github.com/gruntwork-io/terragrunt/config"
+	"github.com/gruntwork-io/terragrunt/internal/errors"
+	"github.com/gruntwork-io/terragrunt/internal/remotestate/backend"
+	"github.com/gruntwork-io/terragrunt/options"
+)
+
+func Run(ctx context.Context, srcPath, dstPath string, opts *options.TerragruntOptions) error {
```

Comment:
NIT: We can replace this with `src` and `dst`.

Reply:
Replaced